### PR TITLE
fix: Use only one font-feature setting

### LINKED ISSUE #1257: Use only one font-feature setting for numbers
Some numbers have ss01 on, some off, which makes them different. I suggest removing `ss01`

### PR DESCRIPTION
**Pull request by beguene**
> Try out this version of the Stacks Wallet - download [extension builds](https://github.com/blockstack/ux/actions/runs/1049078447).

Removed all ss01 from font-settings

See #1257

### CODE REVIEW

**Review by aulneau — APPROVED** [review]
Looks good! Just add a changeset and we're good to go

File: .changeset/cyan-bears-arrive.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@stacks/wallet-web': patch
+---
+
+Fix font feature inconsistency
```
Comment (aulneau) [inline .changeset/cyan-bears-arrive.md]: We use changesets to provide longer form change log messages. Can you take a look at our [current changelog](https://github.com/blockstack/stacks-wallet-web/blob/main/CHANGELOG.md) and update your message to be more verbose? Ideally we'd avoid shorter commit like messages as we have an ability to give more context/references, etc.

Comment (beguene) [reply .changeset/cyan-bears-arrive.md]: Sure I will update that.

**Review by aulneau — APPROVED** [review]

File: .changeset/cyan-bears-arrive.md (hunk 1 above)
Comment (beguene) [reply .changeset/cyan-bears-arrive.md]: Let me know if this is fine, it's my first changeset 😄

**Pull request merged**